### ISSUE TITLE
Freedos COMMAND.COM no longer loads

### ISSUE THREAD

**Issue by severach** [dosemu2/dosemu2#1807]
**Describe the bug**
Loading FreeDOS COMMAND.COM gives the following error.

```
Failed to load the strings resource into memory, the location
pointed to in %COMSPEC% seems to be invalid. Please specify another
location of FreeCOM to try to load the strings from, e.g.:
C:\COMMAND.COM
or just hit enter to cancel to load the strings.
```

**A regression?**
Last working version I have is 4000161551f177865e77f959b0442080d7429c8b 

**Additional info**
Tested on default config

I was using FreeDOS because fdpp/comcom32 was broken on a missing export for a while. The release versions don't work but the git versions do.

**Comment by stsp** [comment]
http://www.ibiblio.org/pub/micro/pc-stuff/freedos/files/repositories/1.3/base/freecom.zip
Putting command.com to drive_c
and works perfectly here.

**Comment by stsp** [comment]
Please see if its probably now fixed.

**Comment by severach** [comment]
command.com from 1.1 and 1.3 both work if C:\command.com is a file not a symlink. Neither work if C:\command.com is a symlink.

My C:\command.com is a symlink. Symlink support for C:\command.com:
Broke by f85645a6ed70a8cf48036e725b6ed7309bf288c5
Fixed by 34d79dfa1d4cd23a910407b50a25f75d92dc2a2f

My kernel.sys is also a symlink. That loaded just fine.
